src/UI/admin/components/hall-reservation/AdminHallReservation.js

```diff
@@ -0,0 +1,272 @@
+                            length={obj.length}
+                            start={obj.start}
+                            exist={obj.exist}
+                            onClick={() => {
+                              console.log(hallReservaltions);
+                              this.setState({
+                                currentReservationActive: hallReservaltions
+                              });
+                            }}
+                          >
+                            <PopUpInfo>
+                              <PopUpInfoText>
+                                {//this.state.dummyDataArray[0].event
+                                //this.state.dummyDataArray[0].hallReservaltions[0]
+                                //.
+                                hallReservaltions.reservationTitle +
+                                  ' ' +
+                                  hallReservaltions.reservationDate +
+                                  ' ' +
+                                  hallReservaltions.reservationStartTime.slice(
+                                    0,
+                                    -3
+                                  ) +
+                                  ' ' +
+                                  hallReservaltions.reservationEndTime.slice(
+                                    0,
+                                    -3
+                                  )}
+                              </PopUpInfoText>
+                            </PopUpInfo>
+                          </ReservationDynamicDiv>
+                        );
+                      }
+                    }
+                  )}
+                </OneDayDivRow>
+              );
+            })}
+          </TableWrapper>
+
+          <Info>
+            <InfoTitle>
+              {
+                //this.state.dummyDataArray[0].event
+                //this.state.dummyDataArray[0].hallReservaltions[0]
+                //.reservationTitle
+                //this.state.currentReservationActive.reservationTitle === undefined ? "WWWWWWWW" : this.state.currentReservationActive.reservationTitle
+                this.state.currentReservationActive.reservationTitle
+                //this.state.currentReservationActive
+              }
+            </InfoTitle>
+            <InfoTime>
+              {//this.state.dummyDataArray[0].startTime.toString().substring(4, 25)
+              this.state.currentReservationActive.reservationDate +
+                ' ' +
+                this.state.currentReservationActive.reservationStartTime.slice(
+                  0,
+                  -3
+                )}{' '}
+              -
+              {//this.state.dummyDataArray[0].endTime.toString().substring(4, 25)
+              this.state.currentReservationActive.reservationEndTime.slice(
+                0,
+                -3
+              )}
+            </InfoTime>
+            <Details
+              img={DetailsIcon}
+              onClick={() => {
```

Comment:
Svu ovu logika unutar onClick event listenera treba odvojiti u posebnu metodu (event handler) koju ćeš pozvati na slj. način `onClick = {this.handleDetailsClick}`